### ISSUE TITLE
Feature Request: More Keyboard Shortcuts, Especially for Copying File + Path

### ISSUE THREAD

**Issue by YMan84** [cboxdoerfer/fsearch#296]
My main use for voidtools Everything, and the main use for using FSearch is to quickly find a file and get its filepath. It would be incredibly useful to have a dedicated keyboard shortcut to copy the "Path + Name". 

Thanks for the fantastic tool!

**Comment by cboxdoerfer** [comment]
Yes, I agree. But I'm not sure if it is a good idea to add too much hard coded shortcuts to the application at this point. I'd much rather wait for the custom shortcuts to be implemented, which are already on my todo list. Then users can use whatever shortcut they prefer for every action available.

However, you should be able to just use `Ctrl+c` and when you then paste into a text field you get the full paths of the selected files.

**Comment by YMan84** [comment]
I didn't think about Ctrl+C, but when i tried it it gives it to me in the following format: 

```
x-special/nautilus-clipboard
copy
file:///home/yuriy/Syncthing/Screenshots/Old/20200221_010054_Discord.png
```
I'll see if I can write some sort of bash script to remove all the unnecessary bits. 
Thanks for the help.

**Comment by cboxdoerfer** [comment]
Ah, I forgot about that. This format was added to support newer versions of Nautilus, because it changed its copy/paste handling. I remember there was quite a controversy about that, so maybe they reverted it, then I can get rid of that format again.
